app/views/subject_viewer.eco

```diff
@@ -2,7 +2,32 @@
 <% moment = require 'moment/moment' %>
 <% Subject = require 'models/experimental_subject' %>
 <% sponsoredCameraTraps = window.SERENGETI_SPONSORED_CAMERA_TRAPS || {} %>
+<% Geordi = require 'lib/geordi_and_experiments_setup' %>
+<% ExperimentServer = Geordi.experimentServerClient %>
 
+<% if ExperimentServer.currentCohort==ExperimentServer.COHORT_UI_EXPERIMENT: %>
```

Comment:
this is the code for the "messaging experiment". It's already live, but perhaps wasn't committed to master